docs/developer/dependencies.rst

```diff
@@ -41,11 +40,21 @@ Run
 Test
 ----
 
-  - all (including optional) dependecies of *Run*
+  - all (including optional) dependencies of *Run*
   - ``mantid-framework`` * (GPL-3.0-or-later) (Mantid interoperability tests are skipped if not present)
   - ``psutil`` (BSD-3-Clause) (used to determine if host has enough memory for certain unit tests)
   - ``pytest`` (MIT) (test framework)
 
+Jupyter Lab
+-----------
+To use scipp with Jupyter Lab:
+
+- ``jupyterlab`` (BSD-3-Clause)
+- ``ipympl`` (BSD-3-Clause) (matplotlib-jupyter integration)
```

Comment:
`ipympl` is needed not just for lab, but also notebook, so should probably be moved above.
Again another example of why it would be good to have all dependencies defined in just one place.